pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -387,6 +387,25 @@ public synchronized PinotResourceManagerResponse deleteSegments(final String tab
     return res;
   }
 
+  /**
+   * Removes deleted segments from the deleted directory
+   * @param deleteRetention: retention for deleted segments
+   * @return
+   */
+  public synchronized PinotResourceManagerResponse removeDeletedSegments(final int deleteRetention) {
```

Comment:
you don't need PinotResourcManagerResponse. A void would do? we are not invoking this via the rest api.